src/mount.ts

```diff
@@ -68,11 +68,18 @@ export type ObjectEmitsOptions = Record<
 export type EmitsOptions = ObjectEmitsOptions | string[]
 
 // Functional component
-export function mount<
-  TestedComponent extends FunctionalComponent<Props>,
-  Props
->(
-  originalComponent: TestedComponent,
+// export function mount<
+//   TestedComponent extends FunctionalComponent<Props, E>,
+//   Props,
+//   E extends EmitsOptions = {}
+// >(
+//   originalComponent: TestedComponent,
+//   options?: MountingOptions<Props>
+// ): VueWrapper<ComponentPublicInstance<Props>>
```

Comment:
Is this commented code still here on purpose? Should it be removed or uncommented? Maybe add an explanation if we want to keep it around.

Reply:
I forgot to remove it

Reply:
I will remove it